pymess/backend/dialer/daktela.py

```diff
@@ -45,25 +47,31 @@ def _update_dialer_states(self, messages):
             resp_message_state = resp_json['result']['statuses'][0]['name'] if len(
                 resp_json['result']['statuses']) else resp_json['result']['action']
             custom_fields = resp_json['result']['customFields']
-            tts_processed = custom_fields['ttsprocessed'][0]
-            whole_message_heard = ('whole_message_heard' in custom_fields and custom_fields['whole_message_heard']
-                                   and custom_fields['whole_message_heard'][0] == 'Yes')
             state_mapped = settings.DIALER_DAKTELA.STATES_MAPPING[resp_message_state]
-            if state_mapped == DialerMessage.STATE.ANSWERED_PARTIAL and whole_message_heard:
-                resp_message_state = str(DialerMessage.STATE.ANSWERED_COMPLETE)
-            if state_mapped == DialerMessage.STATE.DONE and tts_processed == '0':
-                resp_message_state = str(DialerMessage.STATE.NOT_ASSIGNED)
-            message_state = settings.DIALER_DAKTELA.STATES_MAPPING[resp_message_state]
             message_error = resp_json['error'] if len(resp_json['error']) else None
-            tts_processed = resp_json['result']['customFields']['ttsprocessed'][0]
+
+            if message.is_predictive:
```

Comment:
states for the predictive call is simplier

Reply:
nelíbí se mi název prediktivní. To přeci vůbec neříká co to dělá ne? Jedno je navolávání automatické a druhé ruční, možná bych to otočil a dal bych is_automatic_call

Reply:
is_autodialer?

Reply:
good, is_autodialer makes sense. thx

Reply:
renamed